core/wiki/config/ViewTemplateBodyFilters.multids

```diff
@@ -4,7 +4,7 @@ tags: $:/tags/ViewTemplateBodyFilter
 testcase: [tag[$:/tags/wiki-test-spec]type[text/vnd.tiddlywiki-multiple]then[$:/core/ui/TestCaseTemplate]] [tag[$:/tags/wiki-test-spec-failing]type[text/vnd.tiddlywiki-multiple]then[$:/core/ui/TestCaseTemplate]]
 stylesheet: [tag[$:/tags/Stylesheet]then[$:/core/ui/ViewTemplate/body/rendered-plain-text]]
 core-ui-tags: [tag[$:/tags/PageTemplate]] [tag[$:/tags/EditTemplate]] [tag[$:/tags/ViewTemplate]] [tag[$:/tags/KeyboardShortcut]] [tag[$:/tags/ImportPreview]] [tag[$:/tags/EditPreview]][tag[$:/tags/EditorToolbar]] [tag[$:/tags/Actions]] :then[[$:/core/ui/ViewTemplate/body/code]]
-system: [prefix[$:/boot/]] [prefix[$:/config/]] [prefix[$:/core/macros]] [prefix[$:/core/save/]] [prefix[$:/core/templates/]]  [prefix[$:/info/]] [prefix[$:/language/]] [prefix[$:/languages/]] [prefix[$:/snippets/]] [prefix[$:/state/]] [prefix[$:/status/]] [prefix[$:/info/]] [prefix[$:/temp/]] +[!is[image]limit[1]then[$:/core/ui/ViewTemplate/body/code]]
+system: [prefix[$:/boot/]][prefix[$:/core/macros]][prefix[$:/core/save/]][prefix[$:/core/templates/]] [prefix[$:/config/]][prefix[$:/info/]][prefix[$:/language/]][prefix[$:/languages/]][prefix[$:/snippets/]] [prefix[$:/info/]][prefix[$:/state/]][prefix[$:/status/]][prefix[$:/temp/]]:and[!is[image]] :then[[$:/core/ui/ViewTemplate/body/code]]
```

Comment:
Hi @kookma the spaces have been removed from inbetween the filter runs which makes it harder to read the filter and see what has changed